ml-proto/test/memory.wast

```diff
@@ -95,7 +95,7 @@
     (loop
       (if
         (i32.eq (get_local 0) (i32.const 0))
-        (br 1)
+        (br 2)
```

Comment:
if has an implicit label now too. Change was here: 05688003a4d5abf31781141f9b3fde588e454414. Not sure why it didn't break then.